Form/Type/AceEditorType.php

```diff
@@ -9,30 +9,25 @@
  * file that was distributed with this source code.
  */
 
-namespace Norzechowicz\AceEditorBundle\Form\Extension\AceEditor\Type;
+namespace Norzechowicz\AceEditorBundle\Form\Type;
```

Comment:
Sensiolab recommandations https://insight.sensiolabs.com/what-we-analyse/symfony.form.form_type_not_in_type_form_folder